src/main/scala/swiss/dasch/api/BaseEndpoints.scala

```diff
@@ -19,14 +19,22 @@ case class BaseEndpoints(authService: AuthService) {
   val publicEndpoint: PublicEndpoint[Unit, ApiProblem, Unit, Any] = endpoint
     .errorOut(defaultErrorOutputs)
 
+  val maybeSecureEndpoint: ZPartialServerEndpoint[Any, Option[String], Option[Principal], Unit, ApiProblem, Unit, Any] =
```

Comment:
**sugestion**: Can we please use the same name for the conceptually same base endpoint as in dsp-api?